pom.xml

```diff
@@ -6,7 +6,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>f2ea2cb4-c600-4bb5-88e8-e952ff5591ee</groupId>
     <artifactId>mule-vault-properties-providers-module</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>1.3.1-SNAPSHOT</version>
```

Comment:
Version shouldn't have changed, should still be 1.0.0-SNAPSHOT

Reply:
Resolved in the next commit